docs/_articles/en/user-guide/org-browser.md

```diff
@@ -34,25 +34,27 @@ The Org Browser lists all the metadata types **(4)** in your default org. To vie
 You can refresh metadata at org level, for a type, for folders in a type, and for components in a folder. Click refresh icon next to:
 
 - Org Browser **(2)** to refresh available metadata types in the org
-- metadata type **(6)** to get an updated list of components for the type
-- folder metadata type to update the folders in the type, but not the components in the folder
-- folder in a metadata type to refresh the components in it
+- Metadata type **(6)** to get an updated list of components for the type
+- Folder metadata type to update the folders in the type, but not the components in the folder
+- Folder in a metadata type to refresh the components in it
 
-![Metadata type with folders](./images/org_browser_folders.png)
+![Metadata type with folders](../../../images/org_browser_folder_ret.png)
 
 ## Retrieving Metadata
 
 You can retrieve a single metadata component or all components of the same type. Click retrieve icon next to:
 
-- component name **(7)** to retrieve a component to your local project
-- metadata type **(8)** to retrive all components of the type
+- Component name **(7)** to retrieve a component to your local project
+- Metadata type **(8)** to retrieve all components of the type
+
+You can retrieve all folders of folder metadata types such as Reports, Dashboards, Documents, and EmailTemplates, by clicking on the retrieve icon next to the metadata type. When you retrieve a folder, only information about the folder is retrieved  and added to a <folder>-meta.xml file. To retrieve components within folders, view the components in a folder by expanding it and then retrieve the individual component by clicking the retrieve icon. 
 
 Currently, the source is retrieved into your default package directory as defined in the [sfdx-project.json](./en/getting-started/first-project#the-sfdx-projectjson-file) file.
```

Comment:
so, sfdx-project.json should be in courier - so would you put the back ticks within the braces? [`sfdx-project.json`] ??